### ISSUE TITLE
Alphanumeric passcodes

### ISSUE THREAD

**Issue by user879874** [raivo-otp/ios-application#50]
There are currently implementations for biometrics and 6-digit passcode authentication, and they seem to work great.

To help provide users with more freedom in customizing their security level, **alphanumeric passcodes** would be a great idea. It would be the next logical progression in securing users' devices, and it is also frequently a feature for other apps that lock themselves behind passcodes/biometrics.

Would this take long to implement? Other TOTP providers that offer similar authentication already seem to allow alphanumeric passcodes, but not sure how easy those are to implement compared to numerical passcodes.

**Comment by tijme** [comment]
Good idea. It shouldn't take too long to implement. Might implement this when I have the time. Feel free to make a pull request in the mean time!

**Comment by BionicBison05** [comment]
Just had this same thought and noticed this issue hasn't been updated in a while, is this still up for consideration? This would be a useful feature for me.
